engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/spel/Typer.scala

```diff
@@ -28,11 +29,14 @@ import pl.touk.nussknacker.engine.types.EspTypeUtils
 
 import scala.annotation.tailrec
 import scala.reflect.runtime._
-import scala.util.{Success, Failure, Try}
+import scala.util.{Failure, Success, Try}
 import scala.util.control.NonFatal
 
 private[spel] class Typer(classLoader: ClassLoader, commonSupertypeFinder: CommonSupertypeFinder,
-                          dictTyper: SpelDictTyper, strictMethodsChecking: Boolean)(implicit settings: ClassExtractionSettings) extends LazyLogging {
+                          dictTyper: SpelDictTyper, strictMethodsChecking: Boolean,
+                          typeDefinitionSet: TypeDefinitionSet = TypeDefinitionSet(),
```

Comment:
imo kolejnosc typeDefinitionSet i referenceTypeValidating powinna byc odwrotna, to pierwsze ma defaulta

Reply:
fixed